target_postgres/postgres.py

```diff
@@ -70,6 +72,7 @@ def write_batch(self, stream_buffer):
                     versions.add(record_version)
 
                 current_table_schema = self.get_table_schema(cur,
+                                                             (stream_buffer.stream,),
```

Comment:
@AlexanderMann You can use `[ ]` too for list concatenation, if you want to avoid the dangling `,`

Reply:
@awm33 nope, tuples brah. In Python if you wanna create a tuple you just do `(val,)`.

I'm using them like this here to avoid weirdness as we do things further downstream. ie, if we use tuples for these operations, things _should_ behave nicely and will prevent the implementing class from accidentally modifying something.

Reply:
@AlexanderMann Does this have to do with mutability of a tuple vs list?

Reply:
Yeah, it's that and then the comparison of tuples vs lists is much more straightforward. Either way, we can refactor this in the future if folks find this to be far too confusing etc.